### PR TITLE
Lint cases where reftest-wait.js is used but class=reftest-wait isn't

### PR DESCRIPTION
**Pull request by jgraham**

### CODE REVIEW

File: tools/lint/lint.py [hunk 1]
```diff
@@ -655,6 +655,16 @@ def incorrect_path(script, src):
         if incorrect_path("testdriver-vendor.js", src):
             errors.append(rules.TestdriverVendorPath.error(path))
 
+        script_path = None
```
Comment (chrishtr) [inline tools/lint/lint.py:658]: This code checks for unnecessary reftest-wait.js, but what about 
takeScreenshot without reftest-wait?

Comment (jgraham) [reply tools/lint/lint.py:658]: It's checking for reftest-wait.js without reftest-wait, primarilly to find cases where people forgot to add the class, not cases where people included the script needlessly (although there was one of those). Do we have examples of people trying to call takeScreenshot without including the script? That seems less likely.

**Comment by jgraham** [comment]
The Firefox failure is an exisitng instability we already annotated, so I'm happy to merge despite the failure. The underlying problem seems to be loading a SVG image as a list marker; I suspect we don't block the load event on that, and I couldn't make the test stable with a couple of changes I tried.

File: tools/lint/rules.py [hunk 2]
```diff
@@ -197,6 +197,14 @@ class MultipleTestharness(Rule):
     """
 
 
+class MissingReftestWait(Rule):
+    name = "MISSING-REFTESTWAIT"
+    description = "Missing `class=reftest-wait`"
+    to_fix = """
+        ensure tests that will take delayed screenshots use class=reftest-wait on the root element.
```
Comment (chrishtr) [inline tools/lint/rules.py:204]:
```suggestion
        ensure tests that include reftest-wait.js also use class=reftest-wait on the root element.
```

File: tools/lint/lint.py (hunk 1 above)
Comment (chrishtr) [reply tools/lint/lint.py:658]: Ok. I think the to_fix field is incorrect in the other file then, suggested
a rewording.

**Review by chrishtr — APPROVED** [review]
LGTM modulo comments.

**Pull request merged**